### PR TITLE
acquisitions: adds an account serializer

### PR DESCRIPTION
**Pull request by zannkukai**
Adds ES stored data into account serializer.

Co-Authored-by: Renaud Michotte <renaud.michotte@gmail.com>

## Why are you opening this PR?

- Which task/US does it implement?
- Which issue does it fix?

## Dependencies

My PR depends on the following `rero-ils-ui`'s PR(s):

* rero/rero-ils-ui#<xx>

## How to test?

- What command should I have to run to test your PR?
- What should I test through the UI?

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

File: rero_ils/modules/acq_accounts/serializers.py [hunk 1]
```diff
@@ -17,15 +17,38 @@
...
 from ..serializers import JSONSerializer, RecordSchemaJSONV1
 
 
 class AcqAccountJSONSerializer(JSONSerializer):
     """Mixin serializing records as JSON."""
 
+    def preprocess_record(self, pid, record, links_factory=None, **kwargs):
+        """Prepare a record and persistent identifier for serialization."""
+        # Add some ES stored keys into response
+        es_hit = next(AcqAccountsSearch()
```
Comment (rerowep) [inline rero_ils/modules/acq_accounts/serializers.py:34]: If the search has no result `next` will raises an exception. Should we handle this?

File: rero_ils/modules/acq_accounts/serializers.py [hunk 2]
```diff
@@ -35,9 +58,7 @@ def post_process_serialize_search(self, results, pid_fetcher):
             pid = lib_term.get('key')
             if pid not in libraries:
                 libraries[pid] = Library.get_record_by_pid(pid)
```
Comment (rerowep) [inline rero_ils/modules/acq_accounts/serializers.py:60]: We need only the name, why storing the complete library?

**Review by iGormilhit — COMMENTED** [review]
Commit message minor changes:

```
acquisitions: account serializer

* Adds ES stored data into account serializer.
```

- Component at the plural form.
- No trailing dot in the title.

**Review by rerowep — APPROVED** [review]

**Review by iGormilhit — COMMENTED** [review]
Commit message title proposition: ` acquisitions: add an account serializer`

- No trailing dot.
- A verb.

**Pull request merged**